server/rpc.go

```diff
@@ -226,8 +226,8 @@ func startOrchestratorClient(uri *url.URL) (net.OrchestratorClient, *grpc.Client
 		grpc.WithBlock(),
 		grpc.WithTimeout(GRPCConnectTimeout))
 	if err != nil {
-		glog.Error("Did not connect: ", err)
-		return nil, nil, errors.New("Did not connect: " + err.Error())
+		glog.Errorf("Did not connect to %v: %v", uri, err)
```

Comment:
We've started using a `key=value` convention for logging. Maybe something like `"Did not connect to orch=%v err=%v"` . This makes further log analysis slightly easier.

Reply:
Fixed in 8a26ba9